services/web/server/tests/unit/with_dbs/02/test_projects_nodes_handlers__patch.py

```diff
@@ -40,6 +40,15 @@ def mock_project_uses_available_services(mocker: MockerFixture):
     )
 
 
+@pytest.fixture
+def mock_catalog_rpc_get_service(mocker: MockerFixture):
+    mocker.patch(
+        "simcore_service_webserver.projects.projects_api.catalog_rpc.get_service",
+        spec=True,
```

Comment:
You can also raise one of the exceptions in `packages/service-library/src/servicelib/rabbitmq/rpc_interfaces/catalog/errors.py` to check how your code responds to it